### ISSUE TITLE
Build failed: Entry, ":CFBundleIdentifier", Does Not Exist

### ISSUE THREAD

**Issue by BorisB** [ptmt/react-native-macos#172]
os x 10.12.6, xcode 8.3.3 (8E3004b), npm 5.3.0

Right after init new project:

[me@localhost /Volumes/work/MyProject]$ react-native-macos run-macos > report.txt
** BUILD FAILED **
The following build commands failed:
	CompileC /Volumes/work/MyProject/macos/build/Build/Intermediates/React.build/Debug/third-party.build/Objects-normal/x86_64/Bits.o /Volumes/work/MyProject/node_modules/react-native-macos/third-party/folly-2016.09.26.00/folly/Bits.cpp normal x86_64 c++ com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)

The file /Volumes/work/MyProject/macos/build/Build/Products/Debug/MyProject.app does not exist.
Print: Entry, ":CFBundleIdentifier", Does Not Exist

Command failed: /usr/libexec/PlistBuddy -c Print:CFBundleIdentifier build/Build/Products/Debug/MyProject.app/Contents/Info.plist

Print: Entry, ":CFBundleIdentifier", Does Not Exist

[report.txt](https://github.com/ptmt/react-native-macos/files/1211448/report.txt)

**Comment by Songkeys** [comment]
I got the same problem as yours. Several files under `third-party/folly-2016.09.26.00/folly/` are missing.
I have also encountered this problem when using react-native(since 0.45.0) before.
Actually, it is caused by many different factors which will finally result in the same error: `Print: Entry, ":CFBundleIdentifier", Does Not Exist`.
Here is some discussion and solutions but none works for me:
[https://github.com/facebook/react-native/issues/7308](https://github.com/facebook/react-native/issues/7308)
Luckily, what happened to us is quite easy to be solved after my searching for a whole night... I guess you might have use RN old version like me which remains old modules. So simply clear the cache files which contains your missing `folly-2016.09.26.00.tar.gz` file:
```
rm -rf ~/.rncache
```
Now, re-init and re `run-macos`. Works fine~

Hope it helps :)
